ydb/library/yql/core/pg_settings/guc_settings.h

```diff
@@ -13,6 +13,11 @@ class TGUCSettings {
     void Set(const std::string&, const std::string&, bool isLocal = false);
     void Commit();
     void RollBack();
+    std::unordered_map<std::string, std::string> GetSettings() const;
```

Comment:
This class will probably evolve in the future. Please avoid this access to implementation details